providers/squadcast/runbook.go

```diff
@@ -0,0 +1,79 @@
+// service resource is yet to be implemented
+
+package squadcast
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraformutils"
+)
+
+type RunbookGenerator struct {
+	SquadcastService
+	teamID string
```

Comment:
same comment applies everywhere

Reply:
fixed after making use of `Request` function in `squadcast_service.go` file